src/pages/workspace/components/ActivityDetails/ActivityLinks.tsx

```
@@ -60,40 +157,67 @@ const ActivityLinks = ({ viewMode }: ActivityLinksProps) => {
+      {(listLink || []).map((item, index) => (
+        <div style={{ position: 'relative' }}>
+          <Card
+            key={index}
+            hoverable
+            style={{ borderRadius: 12, boxShadow: '0 3px 3px rgba(0, 0, 0, 0.1)' }}
+            styles={{
+              body: { display: 'flex', gap: 16, padding: '10px 0' },
+            }}
+            onClick={() => {
+              if (item.link || '#') {
+                window.open(item.link, '_blank');
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)